README.md

```diff
@@ -125,16 +119,61 @@ If two movies have the same duration, order them in alphabeticall order by their
 
 One of the most famous directors in cinema is **[Steven Spielberg](https://en.wikipedia.org/wiki/Steven_Spielberg)**, and he has some really awesome drama movies that show up on our list, but we want to know how many!
 
-Go ahead and create a `howManyMovies` method that receives an array as a parameter and `filter` :eyes: the array so we can have only the **drama** movies where **Steven Spielberg** is the director. 
+Go ahead and create a `howManyMovies` function that receives an array as a parameter and `filter` :eyes: the array so we can have only the **drama** movies where **Steven Spielberg** is the director. 
 
 
 ### Iteration 6: Alphabetic Order
 
 Another famous way to order the movies is to sort them alphabetically using the `title` key. But in this case we only need to print the title of the first 20. Easy Peasy for an expert like you :wink:
 
-Create a `orderAlphabetically` method, that receive an array and return an array of first 20 titles, alphabetically ordered. Return only the title of each movie, and if the array you receive have less than 20 movies, return all of them order
+Create a `orderAlphabetically` function, that receives an array and return an array of first 20 titles, alphabetically ordered. Return only the title of each movie, and if the array you receive have less than 20 movies, return all of them order
```

Comment:
The sentence wasn't finished and it had minor grammar issue. Please update your file and push the updates.
Suggested update:

Create an `orderAlphabetically` function, that receives an array and returns an array of first 20 titles, alphabetically ordered. Return only the title of each movie, and if the array you receive has less than 20 movies, return all of them ordered alphabetically of course 😉